spec/global_session/session/v4_spec.rb

```diff
@@ -9,7 +9,7 @@
   context 'given an EC key' do
     let(:key_generation_parameter) { 'prime256v1' }
     let(:signature_method) { :dsa_sign_asn1 }
-    let(:approximate_token_size) { 210 }
+    let(:approximate_token_size) { 240 }
```

Comment:
Interesting; I wouldn't expect the token size to change since we're not changing the details of the global session serialization, just fixing a bug in how we verify it.

I wonder if this is because we always set a `scp: null` payload element now, and perhaps the size will shrink back down once we stop doing that?